content/influxdb/v2.2/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,135 @@ menu:
 weight: 101
 ---
 
+## v2.2 [2022-03-29]
+
+This release includes the following new [features](#features) and several [bug fixes](#bug-fixes).
+
+### Features
+
+- [Technical preview of remote data replication](#replicate-data-remotely-technical-preview)
+- [Flux updates](#flux-updates)
+- [Build maintenance](#build-maintenance)
+- [Task metadata](#task-metadata)
+- [Metrics](#metrics)
+- [Display runtime configuration in use by `influxd`](#display-runtime-configuration-in-use-by-influxd)
+- [Recover user credentials](#recover-user-credentials)
+- [Security updates](#security-updates), including a new `hardening-enabled` option to enable additional security.
+
+#### Technical preview: Durable subscription to replicate data
+
+- Add the option to [Replicate data from InfluxDB OSS to InfluxDB Cloud](/influxdb/v2.2/write-data/replication).
+
+  This [**technical preview**](/influxdb/v2.1/reference/glossary/#technical-preview) feature allows you to create a durable subscription on a per bucket basis for the purposes of replicating data from an InfluxDB OSS instance to InfluxDB Cloud. 
```

Comment:
I would just add that it's "replication on write". Like, "...replicating data on-write from an InfluxDB...."